_hackers/mellii.md

```diff
@@ -0,0 +1,5 @@
+---
+fullname: mellii
+twitter: 
+image: mellii.jpg
```

Comment:
@mellii Looks like this doesn't match the image filename (.jpg vs .png) :astonished: